contribute/documentation/README.md

```diff
@@ -0,0 +1,234 @@
+
+While this document doesn't include git commands or descriptions of Github operations, you might find these links useful.
+
+- [Install git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git): We store all source code, including documentation, in Git repositories.
+- [Fork a repo](https://docs.github.com/en/get-started/quickstart/fork-a-repo): Locate the repo you want to clone, and fork it.
+- [Clone a repo](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository): Clone the repository to your local machine.
+- [Create a branch](https://git-scm.com/book/en/v2/Git-Branching-Basic-Branching-and-Merging): Before you make change, create a branch. Do not push changes against the `main` branch.
+- [Create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request): After you add, commit, and push your changes, create a PR in Github.
+
+#### Use a documentation template to contribute a topic
+
+We have provided documentation templates that align with each topic type:
+
+- [Concept](../templates/doc-concept-template.md)
+- [Task](../templates/doc-task-template.md)
+- [Reference](../templates/doc-reference-template.md)
+
+Each template provides additional usage and formatting guidelines. We recommend that you make a copy of whichever template you are using, and add content.
+
+:::note
+Remove any unused content before you commit your changes.
+:::
+
+#### View a local build
+
+Prior to pushing your changes, you can view a local build of the documentation so that you can review your work.
+
+To view a local build:
+
+1/ Install [Docker](https://www.docker.com/products/docker-desktop).
+
+1/ Run Docker.
+
+1/ Navigate to the **docs** root directory.
+
+1/ Run `make build`.
+
+1/ Open `localhost:3002` to review your changes.
+
+## Push changes and create a PR
+
+When you are ready for other people to review your work, perform the following tasks.
+
+1/ [Add](https://git-scm.com/docs/git-add) your changes, which prepares your content for the next commit.
+
+1/ [Commit](https://git-scm.com/docs/git-commit) your changes.
+
+1/ [Push](https://git-scm.com/docs/git-push) your changes to Github.
+
+1/ [Create a PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) in Github.
+
+   The docs build system automatically conducts a series of tests to ensure that the content doesn't conflict with other content in the docs repository.
+
+### Understanding the PR review and approval workflow
+
+Need content here that describes where to post PR for review, of whom to select for a review, review SLA?, and what the review process is like. This doesn't have to be long.
```

Comment:
Taking a stab here (@achatterjee-grafana may want to further refine):
```suggestion
When a PR is added to the repo with a label associated with `docs`, it will be reviewed by a member of the technical writing team. Depending on the size of the PR and the priority of other work, the PR will either be immediately reviewed and merged (minor fixes typically follow this pattern) or the PR will be triaged and placed in the backlog of work or moved into further development.
```